test/item-processor-test.js

```diff
@@ -43,9 +43,10 @@ test('processor', {
     assert.equal(this.item.dates.fullDate.created, 'January 1st 1970');
     assert.equal(this.item.dates.fullDate.modified, 'January 2nd 1970');
     assert.equal(this.item.dates.fullDate.rendered, 'January 3rd 1970');
-    assert.equal(this.item.dates.someTime.created, timezoneOffSet+':00:00');
-    assert.equal(this.item.dates.someTime.modified, (timezoneOffSet+1)+':00:00');
-    assert.equal(this.item.dates.someTime.rendered,  (timezoneOffSet+2)+':00:00');
+    // zero-pad, ref: https://groups.google.com/forum/#!topic/nodejs/df5UzXFAByA
+    assert.equal(this.item.dates.someTime.created, ("0" + timezoneOffSet).slice(-2)+':00:00');
+    assert.equal(this.item.dates.someTime.modified, ("0" + (timezoneOffSet+1)).slice(-2) +':00:00');
+    assert.equal(this.item.dates.someTime.rendered, ("0" + (timezoneOffSet+2)).slice(-2)+':00:00');
```

Comment:
Drats, sorry! I finally looked up how to test with any TZ so will do a new PR to fix this.